client/src/views/NewEventTypeAdditionalOptions.vue

```diff
@@ -0,0 +1,170 @@
+                                </VCol>
+                                <VCol cols="10" class="pl-lg-5 pl-sm-10">
+                                    <div>
+                                        <VCardTitle>
+                                            {{ lang.ADDITIONAL_OPTIONS }}
+                                        </VCardTitle>
+                                        <VCardSubtitle>
+                                            {{ lang.QUESTIONS_TO_INVITEE }}
+                                        </VCardSubtitle>
+                                    </div>
+                                </VCol>
+                            </VRow>
+                        </VExpansionPanelHeader>
+                        <VExpansionPanelContent>
+                            <VDivider class="mx-4"></VDivider>
+                            <VRow>
+                                <VCol
+                                    cols="10"
+                                    offset-sm="2"
+                                    offset-md="2"
+                                    md="7"
+                                    sm="6"
+                                    lg="7"
+                                    :class="{ 'ml-10': $vuetify.breakpoint.xs }"
+                                >
+                                    <CreateEventTypeQuestions
+                                        :eventTypeId="$route.query.eventTypeId"
+                                    />
+                                </VCol>
+                            </VRow>
+                        </VExpansionPanelContent>
+                    </VExpansionPanel>
+                </VExpansionPanels>
+            </VCard>
+        </VContainer>
+        <template v-slot:title>
+            <NewEventTypeTitle />
+        </template>
+    </GeneralLayout>
+</template>
+
+<script>
+import GeneralLayout from '@/components/common/GeneralLayout/GeneralLayout';
+import NewEventTypeTitle from '@/components/events/NewEventTypeTitle';
+import CreateEventTypeBookingForm from '@/components/events/CreateEventTypeBookingForm';
+import FirstTitle from '@/components/events/CreateEventTypeTitle';
+import CreateEventTypeForm from '@/components/events/CreateEventTypeForm';
+import eventTypeMixin from '@/components/events/eventTypeMixin';
+import CreateEventTypeQuestions from '@/components/events/custom-fields/CreateEventTypeQuestions';
+export default {
+    name: 'NewEventTypeAdditionalOptions',
+    mixins: [eventTypeMixin],
+    components: {
+        GeneralLayout,
+        NewEventTypeTitle,
+        CreateEventTypeBookingForm,
+        FirstTitle,
+        CreateEventTypeForm,
+        CreateEventTypeQuestions
+    },
+    data() {
+        return {
+            colorById: ''
+        };
+    },
+    computed: {
+        duration() {
+            return (
+                (this.eventType.customDuration || this.eventType.duration) +
+                ' ' +
+                this.lang.MIN
```

Comment:
The variable is used in length validation (and has wrong Ukrainian translation). `DURATION_MIN` should be used here.